saleor/graphql/webhook/mutations.py

```diff
@@ -419,3 +425,115 @@ def perform_mutation(cls, _root, info, **data):
+            raise_validation_error(
+                message="Can't parse an event type from webhook's subscription query.",
+                code=WebhookTriggerErrorCode.UNABLE_TO_PARSE,
+            )
+
+        event = WEBHOOK_TYPES_MAP.get(event_type) if event_type else None
+        if not event and event_type:
+            event_name = event_type[0].upper() + to_camel_case(event_type)[1:]
+            raise_validation_error(
+                message=f"Event type: {event_name}, which was parsed from webhook's "
+                f"subscription query, is not defined in graphql schema.",
+                code=WebhookTriggerErrorCode.GRAPHQL_ERROR,
+            )
+
+        model, _ = graphene.Node.from_global_id(object_id)
+        model_name = event._meta.root_type  # type: ignore[union-attr]
+        enable_dry_run = event._meta.enable_dry_run  # type: ignore[union-attr]
+
+        if not (model_name or enable_dry_run) and event_type:
+            event_name = event_type[0].upper() + to_camel_case(event_type)[1:]
+            raise_validation_error(
+                message=f"Event type: {event_name}, which was parsed from webhook's "
+                f"subscription query, is not supported.",
+                code=WebhookTriggerErrorCode.TYPE_NOT_SUPPORTED,
+            )
+        if model != model_name:
+            raise_validation_error(
+                field="objectId",
+                message="ObjectId doesn't match event type.",
+                code=WebhookTriggerErrorCode.INVALID_ID,
+            )
+
+        if (
+            permission := WebhookEventAsyncType.PERMISSIONS.get(event_type)
+            if event_type
+            else None
+        ):
+            codename = permission.value.split(".")[1]
+            user_permissions = [
+                perm.codename for perm in info.context.user.effective_permissions.all()
+            ]
+            if codename not in user_permissions:
+                raise_validation_error(
+                    message=f"The user doesn't have required permission: {codename}.",
+                    code=WebhookTriggerErrorCode.MISSING_PERMISSION,
+                )
+
+        object = cls.get_node_or_error(info, object_id, field="objectId")
+
+        return event_type, object, webhook
+
+    @classmethod
+    def perform_mutation(cls, _root, info, **data):
+        from ...plugins.webhook.tasks import (
+            create_deliveries_for_subscriptions,
+            send_webhook_request_async,
+        )
+
+        event_type, object, webhook = cls.validate_input(info, **data)
+        delivery = None
+
+        if all([event_type, object, webhook]):
+            deliveries = create_deliveries_for_subscriptions(
+                event_type, object, [webhook]
+            )
+            if deliveries:
+                delivery = deliveries[0]
+                try:
+                    send_webhook_request_async(delivery.id)
+                    return WebhookTrigger(delivery=None)
```

Comment:
We could do:
```
return WebhookTrigger(delivery=delivery)
```
It would return delivery object with at least some of the fields set. It doesn't need all of them eg payload or attempts if they are removed from DB.